docs/element-definitions.md

```diff
@@ -1118,6 +1118,53 @@ Data structure definition using Data Structure elements.
 - `"messageBody"` - Asset is an example of message-body
 - `"messageBodySchema"` - Asset is a schema for message-body
 
+### Server ([Object][])
+
+Subtype of [Object][] representing a server.
+
+#### Template
+
+- `element` - `"server"`
+- `attributes`
+    - `href` ([Templated Href][]) - URI Template for this server.
+    - `hrefVariables` ([Href Variables][]) - URI Template variables.
+
+#### Example
+
+```json
+{
+  "element": "server",
+  "meta": {
+    "description": {
+      "element": "string",
+      "content": "Description of the server."
+    }
+  },
+  "attributes": {
+    "href": {
+      "element": "string",
+      "content": "{basePath}://example.com/"
+    },
+    "hrefVariables": {
+      "element": "hrefVariables",
+      "content": [
+        {
+          "element": "member",
+          "content": {
+            "key": {
+              "element": "string",
+              "content": "basePath"
+            }
```

Comment:
```suggestion
            },
            "value": {
              "element": "string",
              "content": "https"
            }
```

It would be good to include an example of the scheme since OAS 3 will usually have an example too (I think its required in OAS 3 to provide default/example value).